setup.py

```diff
@@ -173,7 +173,7 @@ def get_git_sha() -> str:
         "impala": ["impyla>0.16.2, <0.17"],
         "kusto": ["sqlalchemy-kusto>=2.0.0, <3"],
         "kylin": ["kylinpy>=2.8.1, <2.9"],
-        "mssql": ["pymssql>=2.1.4, <2.2"],
+        "mssql": ["pymssql>=2.2.8"],
```

Comment:
Would you mind re-including an major version upper bound to prevent potential regressions due to major version bumps?

Reply:
@john-bodley thx for your advise, fixed.